build-docker.sh

```diff
@@ -19,10 +19,11 @@ if [ -z "$ALPINE_ARCH" ]; then
 fi
 
 CONTAINER_NAME=${CONTAINER_NAME:-trezor-firmware-env.nix}
-ALPINE_CDN=${ALPINE_CDN:-http://dl-cdn.alpinelinux.org/alpine}
+ALPINE_CDN=${ALPINE_CDN:-https://dl-cdn.alpinelinux.org/alpine}
 ALPINE_RELEASE=${ALPINE_RELEASE:-3.14}
 ALPINE_VERSION=${ALPINE_VERSION:-3.14.0}
 ALPINE_TARBALL=${ALPINE_FILE:-alpine-minirootfs-$ALPINE_VERSION-$ALPINE_ARCH.tar.gz}
+ALPINE_CHECKSUM=${ALPINE_CHECKSUM:-bcdf5a4e58637b9228f8e474547a3de9ea02a05a5fa68a2495b0657aaa7e65f6}
```

Comment:
This will break on `aarch64`. Let's use the similar trick as above:

```
if [ -z "$ALPINE_CHECKSUM" ]; then
  case "$ALPINE_ARCH" in
    aarch64)
      ALPINE_CHECKSUM ="bc541e148463b3dde10fdbb1af8eac4e34706eae8883c6d126263db07a9a9c42"
      ;;
    x86_64)
      ALPINE_CHECKSUM ="bcdf5a4e58637b9228f8e474547a3de9ea02a05a5fa68a2495b0657aaa7e65f6"
      ;;
    *)
      echo "Unsupported arch"
      exit
  esac
 fi
```

Reply:
You're right didn't think of that. Pushed commit with your suggestion, and it should work as expected.